<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExampleGui</class>
 <widget class="QWidget" name="ExampleGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RCON Test</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="disconnectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="portInput">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>2302</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="passwordInput">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="commandButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Command</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>IP/Host</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="hostInput"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="commandInput">
     <property name="text">
      <string>players</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTextBrowser" name="logOutput"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
